src/main/resources/application.yml

```diff
@@ -0,0 +1,25 @@
+spring:
+  datasource:
+    url: ${SPRING_DATASOURCE_URL}
+    username: ${SPRING_DATASOURCE_USERNAME}
+    password: ${SPRING_DATASOURCE_PASSWORD}
+    driver-class-name: com.mysql.cj.jdbc.Driver
+
+  jpa: 
+    hibernate:
+      ddl-auto: update
+  redis: 
+    lettuce:
+      pool: 
+        max-active: 10
+        max-idle: 10
+        min-idle: 2
+      port: 6379
+      host: 127.0.0.1
```

Comment:
You've already define the port and host above, just like hard-coding-way
Why did you make env here also?
How about use @Value in code when you use envs?
And this should be define also in docker-compose.yml